ci/docker_ci.sh

```diff
@@ -85,9 +113,19 @@ push_images() {
   _args=$(build_args "${TYPE}")
   read -ra args <<<"$_args"
   PLATFORM="$(build_platforms "${TYPE}")"
+
+  echo ">> PUSH: ${BUILD_TAG}"
+  echo "> docker ${BUILD_COMMAND[*]} --platform ${PLATFORM} ${args[*]} -t ${BUILD_TAG} . --push ||
+    docker push ${BUILD_TAG}"
+  if [[ -n "$DOCKER_CI_DRYRUN" ]]; then
+      echo ""
+      return
+  fi
+  echo "..."
   # docker buildx doesn't do push with default builder
   docker "${BUILD_COMMAND[@]}" --platform "${PLATFORM}" "${args[@]}" -t "${BUILD_TAG}" . --push ||
     docker push "${BUILD_TAG}"
+  echo ""
```

Comment:
is this necessary?

Reply:
it makes it easier to parse the output - ie it separates the blocks